extensions/wikia/CommunityPage/styles/components/_variables.scss

```diff
@@ -8,6 +8,8 @@ $community-page-font-family: "HelveticaNeue", Helvetica, Arial, sans-serif;
 $module-default-line-height: 20px;
 $avatar-size: 30px;
 $avatar-flex: 0 0 $avatar-size;
+$username-column-width: 174px;
+$userrank-column-width: 26px;
```

Comment:
By that comment, I meant that `userrank-column-width` is not actual size, it is just a difference between two different username sizes. Do you think we could use less `artificial` variables?